src/lib/types.ts

```diff
@@ -33,6 +33,16 @@ export interface SWIFTCreditPaymentInstruction extends PaymentInstruction {
   creditor: Party;
 }
 
+/*
+ * Represents a SEPA credit payment instruction, etending the base PaymentInstruction.
+ */
+export interface SEPACreditPaymentInstruction extends PaymentInstruction {
+  type: 'sepa',
+  direction: 'credit',
+  creditor: Party,
+  dinero?: Dinero.Dinero,
```

Comment:
Is there a way we can get rid of Dinero as a parameter here? It seems nice to have this package abstracted away from the iso20022.js user

Reply:
Thanks to our new changes we can remove this `dinero?`

Reply:
In principle, yes, since they are now generated internally. I'll make that commit.